Tests/ASDisplayNodeTests.mm

```diff
@@ -1922,18 +1923,23 @@ - (void)checkBackgroundColorOpaqueRelationshipWithViewLoaded:(BOOL)loaded layerB
   }
 
   XCTAssertTrue(node.opaque, @"Node should start opaque");
-  XCTAssertTrue(node.layer.opaque, @"Node should start opaque");
+  if (isLayerBacked) {
+      XCTAssertTrue(node.layer.opaque, @"Set background color should not have made this layer not opaque");
+    } else {
+      XCTAssertTrue(node.view.opaque, @"Set background color should not have made this view not opaque");
+    }
```

Comment:
Is this necessary? At the beginning the view and the layer should agree with each other?

Nit: indentation is off.

Reply:
gah 2 spaces

Reply:
so they don't agree with each other in practice. i'll add a comment here